x-pack/plugins/endpoint/common/models/event.ts

```diff
@@ -46,3 +47,42 @@ export function parentEntityId(event: ResolverEvent): string | undefined {
   }
   return event.process.parent?.entity_id;
 }
+
+export function eventCategoryDisplayName(event: ResolverEvent): EventCategory {
+  const eventTypeToNameMap = new Map<string, EventCategory | undefined>([
+    ['process', 'Process'],
```

Comment:
if these are meant to printed out/displayed they need to be i18n'd

Reply:
Thanks for the 👁️ . https://github.com/elastic/kibana/pull/64121/commits/7bb0c308d0631cdcb0501c24f692cd4abbaba8f5